src/main/java/seedu/address/ui/panel/log/LogBox.java

```diff
@@ -18,8 +19,9 @@
      * Constructor for LogBox. It is a dialog box that contains the feedback from the program to the user.
      * @param feedbackToUser The feedback from the program to the user.
      */
-    public LogBox(String feedbackToUser) {
+    public LogBox(String feedbackToUser, String color) {
         super(FXML);
         box.setText(feedbackToUser);
+        box.setStyle("-fx-background-color: " + color);
```

Comment:
This works, but just letting you know that javafx allows you to set the background color without using css :)
`label.setBackground(new Background(new BackgroundFill(color, CornerRadii.EMPTY, Insets.EMPTY)));`